### PR TITLE
Scala 2.13 support in app

### PR DESCRIPTION
**Pull request by jtjeferreira**
[ ] https://github.com/dispatch/reboot/issues/210

### CODE REVIEW

**Comment by jtjeferreira** [comment]
I think this is blocked on https://github.com/dispatch/reboot/issues/210

**Comment by eed3si9n** [comment]
@jtjeferreira Thanks for this contribution. I am looking forward to getting green. Hopefully Dispatch will get 2.13 soon now that Unfiltered is out? http://repo1.maven.org/maven2/ws/unfiltered/unfiltered_2.13/0.10.0-M4/

**Comment by MariosPapasofokli** [comment]
@jtjeferreira @eed3si9n Dispatch/reboot has a new release for Scala 2.13 

- https://github.com/dispatch/reboot/releases
- https://repo1.maven.org/maven2/org/dispatchhttp/dispatch-core_2.13/

File: integration/src/test/scala-2.13+/CompilerMatcher.scala [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+      import scala.tools.nsc.interpreter.{IMain, Results => IR}
+      import scala.util.control.Exception._
+
+      val code = pair.value._1
+      val files = pair.value._2
+      if (code.size < 1)
+        sys.error("At least one line of code is required.")
+      val s = settings(outdir, classpath, usecurrentcp, unchecked,
+        deprecation, feature, fatalWarnings)
+      val main = new IMain(s, new scala.tools.nsc.interpreter.shell.ReplReporterImpl(s)) {
```
Comment (jtjeferreira) [inline integration/src/test/scala-2.13+/CompilerMatcher.scala]: I had to duplicate this file, because of the change is this line

**Pull request merged**